Gemfile.lock

```diff
@@ -81,17 +81,20 @@ GEM
     bundler-audit (0.3.1)
       bundler (~> 1.2)
       thor (~> 0.18)
-    byebug (6.0.2)
+    byebug (2.7.0)
+      columnize (~> 0.3)
+      debugger-linecache (~> 1.2)
     code_analyzer (0.4.4)
       sexp_processor
     colored (1.2)
+    columnize (0.3.6)
     concurrent-ruby (1.0.0)
     crack (0.4.2)
       safe_yaml (~> 1.0.0)
-    daemons (1.1.9)
     database_cleaner (1.2.0)
+    debugger-linecache (1.2.0)
     decent_exposure (2.3.1)
-    devise (3.5.6)
+    devise (3.5.1)
```

Comment:
The version is downgraded
See https://semaphoreci.com/fs/rails-base-api/branches/pull-request-187/builds/2